lib/Cron/Workers/SendingQueue/SendingQueue.php

```diff
@@ -279,6 +308,29 @@ public function sendNewsletters(
     );
   }
 
+  /**
+   * Checks whether some of segments was deleted or trashed
+   * @param int[] $segmentIds
+   * @return bool
+   */
+  private function checkDeletedSegments(array $segmentIds) {
```

Comment:
Is there a reason not to specify the returned type? `private function checkDeletedSegments(array $segmentIds): boolean`